### PR TITLE
Create misinformation-disinformation-plaguing-online-platforms-before…

### PR DESCRIPTION
**Pull request by realjenny**
…-house-energy-commerce-committee.en.md

### CODE REVIEW

File: frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: 'Statement of the Alphabet Workers Union On “the Misinformation and Disinformation Plaguing Online Platforms” Before the House Energy and Commerce Committee March 25, 2021s'
```
Comment (justaliu) [inline frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md:2]:
```suggestion
title: 'Statement of the Alphabet Workers Union On “the Misinformation and Disinformation Plaguing Online Platforms” Before the House Energy and Commerce Committee March 25, 2021'
```

**Review by justaliu — APPROVED** [review]

File: frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md (hunk 1 above)
Comment (justaliu) [reply frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md:2]: also, not sure if you want to add a comma between "Committee" and "March"

File: frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+8. [YouTube’s purge of white supremacist videos also hits anti-racism channels](https://www.latimes.com/business/la-fi-tn-youtube-hate-extremism-20190606-story.html) from the LA Times
+9. [The Christchurch Shooter and YouTube’s Radicalization Trap](https://www.wired.com/story/christchurch-shooter-youtube-radicalization-extremism/) from WIRED
+10. [YouTube faces complaints of lax approach on overseas election misinformation](https://www.reuters.com/article/us-youtube-myanmar-misinformation/youtube-faces-complaints-of-lax-approach-on-overseas-election-misinformation-idUSKBN28S0QE) from Reuters
+11. [Auditing radicalization pathways on YouTube](https://dl.acm.org/doi/abs/10.1145/3351095.3372879)
+12. [How QAnon and other dark forces are radicalizing Americans as the COVID-19 pandemic rages and election looms](https://www.usatoday.com/in-depth/tech/2020/08/31/qanon-conspiracy-theories-trump-election-covid-19-pandemic-extremist-groups/5662374002/) From USA Today
```
Comment (justaliu) [inline frontend/content/press/releases/misinformation-disinformation-plaguing-online-platforms-before-house-energy-commerce-committee.en.md:50]:
```suggestion
12. [How QAnon and other dark forces are radicalizing Americans as the COVID-19 pandemic rages and election looms](https://www.usatoday.com/in-depth/tech/2020/08/31/qanon-conspiracy-theories-trump-election-covid-19-pandemic-extremist-groups/5662374002/) from USA Today
```

**Review by agdphd — APPROVED** [review]
Does this Markdown engine support footnote links by number? If so that might be better than duplicating the link URLs.

**Review by butterflyhug — APPROVED** [review]
Probably not essential but the automated check suggests running Prettier to standardize formatting.

**Review by Gdakgdak22 — APPROVED** [review]

**Pull request merged**